File: 백준/Bronze/34027. 제곱 수？/제곱 수？.c
#include <math.h>
#include <stdio.h>
int main(void) {
    int t, n;
    double tmp;
    scanf("%d", &t);
    for (int i = 0; i < t; i++) {
        scanf("%d", &n);
        tmp = sqrt(n);
        printf("%d\n", tmp == (int)tmp ? 1 : 0);
    }
    return 0;
}

File: 백준/Bronze/34027. 제곱 수？/제곱 수？.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.lang.Math;
public class Main {
    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n,t = Integer.parseInt(br.readLine());
        double tmp;
        for(int i=0;i<t;i++) {
            n = Integer.parseInt(br.readLine());
            tmp = Math.sqrt(n);
            System.out.println(tmp==(int)tmp?1:0);
        }
    }
}